pages/exercise/AnnouncementHandler.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+/**
+ * @file pages/about/AboutHandler.php
+ *
+ * Copyright (c) 2014-2021 Simon Fraser University
```

Comment:
Not a big deal -- we're not 100% consistent with this -- but for new code, it should be (c) the current year:
```
 * Copyright (c) 2023 Simon Fraser University
 * Copyright (c) 2023 John Willinsky
```